dali/operators/reader/loader/numpy_loader.cc

```diff
@@ -69,15 +70,18 @@ void NumpyLoader::ReadSample(NumpyFileWrapper& target) {
     return;
   }
 
-  auto path = filesystem::join_path(file_root_, filename);
-  auto current_file = FileStream::Open(path, read_ahead_, !copy_read_data_, use_o_direct_);
+  auto uri = filesystem::join_path(file_root_, filename);
```

Comment:
This is a bit smelly. `filesystem::join_path` will use the local file system's directory separator - meanwhile, we might be constructing an URI, which doesn't need to be local and should use `/`. This is purely theoretical, since now we support only UNIX, but it's a readily waiting bug should we support Windows natively at some point.